scalalib/src/mill/scalalib/JavaModule.scala

```diff
@@ -1056,14 +1056,15 @@ trait JavaModule
     canRun = true
   )
 
+  @internal
+  def jvmBuildTarget: JvmBuildTarget =
```

Comment:
Should we call this `bspJvmBuildTarget`, just so it's clear it's related to the other `bsp*` members